.gitignore

```diff
@@ -3,3 +3,5 @@
 /**/dbt_packages/
 /**/logs/
 /**/env/
+/**/.idea/
```

Comment:
nit: Maybe better to add to a global .gitignore instead of here.

Reply:
removed from here